app/src/main/java/com/android/wildex/ui/achievement/AchievementsScreen.kt

```diff
@@ -27,7 +175,36 @@ fun AchievementsScreen(viewModel: AchievementsScreenViewModel, onGoBack: () -> U
  * @param achievement The [Achievement] to display, including its name and image URL.
  * @param unlocked Whether the achievement has been unlocked by the user.
  */
-@Composable fun AchievementItem(achievement: Achievement, unlocked: Boolean) {}
+@Composable
+fun AchievementItem(achievement: Achievement, unlocked: Boolean) {
+  Column(
+      horizontalAlignment = Alignment.CenterHorizontally,
+      modifier = Modifier.width(90.dp).testTag(AchievementsScreenTestTags.ACHIEVEMENT_ITEM)) {
+        Image(
```

Comment:
I think it might be better to use an `AsyncImage` object here since we load the image from Firebase. It would be better since it allows asynchronous loading.
```suggestion
        AsyncImage(
```